### PR TITLE
Editorial suggestions from Benjamin Kaduk's IESG review (priority)

### PR DESCRIPTION
**Pull request by kaduk**

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -427,7 +427,8 @@ where to send registration requests.
 
 The Priority HTTP header field carries priority parameters (see {{parameters}}).
 It can appear in requests and responses. It is an end-to-end signal of the
-request priority from the client or the response priority from the server.
+requested response
```
Comment (kaduk) [inline draft-ietf-httpbis-priority.md:430]: The overall focus seems to be on the priority as being about the priority of the response, even though the header field can appear in the request.  But the origin is authoritative on the actual response priority, so it feels weird to say "response priority from the client".

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -825,8 +826,8 @@ those coming from others.
 
 It is sometimes beneficial for the server running behind an intermediary to obey
 Priority header field values. As an example, a resource-constrained
-server might defer the transmission of software update files that would have the
-background urgency being associated. However, in the worst case, the asymmetry
+server might defer the transmission of software update files that have the
+background urgency. However, in the worst case, the asymmetry
```
Comment (kaduk) [inline draft-ietf-httpbis-priority.md:830]: Not sure if this is the right fix, but something seemed off here and this is a proposal for it.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:430]: Overrall the draft is attempting to articulate the client's view of the urgency/incremental based on what the client knows locally. It's half the picture and the client cannot do anything about what actually happens. This is not like, for example content negotiation or anything. At the time the header is consumed (say by an intermediary) there may be no response so to speak of. 

The term "requested response" feels weird to me given this document and HTTP semantics more broadly

**Comment by LPardue** [comment]
Thanks @kaduk . I've responded the lines you've commented on. The other editorial changes all look good to me.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (kaduk) [reply draft-ietf-httpbis-priority.md:430]: My local notes (prior to conversion to PR) prefaced this remark with something like "would we be willing to consider" and a section reference to section 3 to explain some of the intended motivation.
I'll attempt to ponder if I want to make a different suggestion or just drop it, over the next couple days.

(And to be clear: the intended term was "requested response priority" but I put in an arbitrary line break to keep any CI that looks at line lengths from complaining.  I think your point stands, regardless.)

Comment (MikeBishop) [reply draft-ietf-httpbis-priority.md:430]: Perhaps reword the sentence more broadly -- "It is an end-to-end signal of each endpoint's local view of the response priority."

**Review by LPardue — CHANGES_REQUESTED** [review]

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:430]: I like that @MikeBishop. Although I might tweak it a bit to something like

"It is an end-to-end signal that indicates the endpoint's view on how HTTP responses should be prioritized."

File: draft-ietf-httpbis-priority.md (hunk 2 above)
Comment (LPardue) [reply draft-ietf-httpbis-priority.md:830]: Taking a closer look, this seems like text that survived from some early drafts and not something that got botched by any subsequent edits. @kaduk's suggestion here works for me.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md:430]: That SGTM.

File: draft-ietf-httpbis-priority.md (hunk 2 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md:830]: +1 to adopting the suggested change.

**Review by kazuho — COMMENTED** [review]
@kaduk Thank you for the PR.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (kaduk) [reply draft-ietf-httpbis-priority.md]: I pushed a new commit with (a one-word tweak of) Lucas' version.  Thanks for the improved suggestion!

**Review by LPardue — APPROVED** [review]
LGTM now. Thank you for these suggestions @kaduk!

**Pull request merged**